src/task_scl_structural_habitat.py

```diff
@@ -32,337 +37,49 @@ def __init__(self, *args, **kwargs):
         )
         self.elevation = ee.Image(self.inputs["elevation"]["ee_path"])
         self.zones = ee.FeatureCollection(self.inputs["zones"]["ee_path"])
-        # self.set_aoi_from_ee(self.inputs["probability"]["ee_path"])
+        self.lc_elev_reclass = ee.FeatureCollection(
+            self.inputs["lc_elev_reclass"]["ee_path"]
+        )
 
     def calc(self):
 
-        zones_img = self.zones.reduceToImage(
-            properties=['Biome_zone'],
-            reducer=ee.Reducer.mode()
-        ).rename('Biome_zone')
-
-        def remap_elev(from_list, to_list, zone_number, z_img, elev_img, lc_img):
-            return (elev_img.lt(
-            lc_img.remap(from_list, to_list)
-                .updateMask(z_img.eq(zone_number)))
-                .multiply(100))
-
+        zone_numbers = self.zones.aggregate_histogram("Biome_zone").keys()
```

Comment:
It wouldn't hurt to do 
`BIOME_ZONE_LABEL = "Biome_zone"` (constant) up with the other class properties, i.e. near `scale` etc. Same with, say, `ELEV_ZONE_LABEL = "elev_zone"` and so on, and then replace below.

Reply:
that make sense.  will do.